packages/ui/src/app/pages/Settings/Settings.tsx

```diff
@@ -204,7 +223,7 @@ export const Settings = () => {
                           /^(ws|wss):\/\//i.test(customRpcEndpoint)
                             ? isValidRpcEndpoint
                               ? undefined
-                              : 'Connection Error'
+                              : 'Connection Error, Sometimes it failed by network speed, Please try to check once more'
```

Comment:
```suggestion
                              : 'Connection Error. Sometimes it fails due to network speed. Please try to check once more'
```

Reply:
Thanks for your detailed suggestion. I learned a lot.

Reply:
No problem!